App/features/bootstrap/FeatureContext.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+use Behat\Behat\Context\Context;
+use Behat\Gherkin\Node\PyStringNode;
+
+class FeatureContext implements Context
+{
+
+    private string $output;
+
+    private function setOutput($output)
+    {
+        $this->output = $output;
+    }
+
+    private function getOutput()
+    {
+        return $this->output;
+    }
+
+    /**
+     * @Given /^I am in a directory "([^"]*)"$/
+     */
+    public function iAmInADirectory($dir)
```

Comment:
On peut aussi écrire ça comme ça. Pas besoin de regex pour un cas aussi simple, on peut nommer les paramètres et les typer dans la signature
```suggestion
    /**
     * @Given I am in a directory ":dir"
     */
    public function iAmInADirectory(string $dir)
```